controllers/bsl.go

```diff
@@ -102,12 +102,29 @@ func (r *DPAReconciler) ReconcileBackupStorageLocations(log logr.Logger) (bool,
 		// 	 2. <namespace>.dataprotectionapplication: <name>
 		// which in turn will be used in th elabel handler to trigger the reconciliation loop
 
-		secretName, _ := r.getSecretNameAndKeyforBackupLocation(bslSpec)
+		secretName, secretKey := r.getSecretNameAndKeyforBackupLocation(bslSpec)
```

Comment:
Maybe we should move this to `ValidateDataProtectionCR` [function](https://github.com/openshift/oadp-operator/blob/22d063a852175a5d89835d856f450c9ae596965c/controllers/dpa_controller.go#L95) ?

Reply:
Could do that too

Reply:
Done and verified.